pages/api/admin/connections/index.ts

```diff
@@ -0,0 +1,121 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+import jackson from '@lib/jackson';
+import { strategyChecker } from '@lib/utils';
+import { checkSession } from '@lib/middleware';
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  const { method } = req;
+
+  switch (method) {
+    case 'GET':
+      return handleGET(req, res);
+    case 'POST':
+      return handlePOST(req, res);
+    case 'PATCH':
+      return handlePATCH(req, res);
+    case 'DELETE':
+      return handleDELETE(req, res);
+    default:
+      res.setHeader('Allow', ['GET', 'POST', 'PATCH', 'DELETE']);
```

Comment:
Setting an array here would send multiple `Allow` headers in the response (here and in other api routes).
The standard approach is to set a comma separated list https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Allow.

Reply:
Agree. Better address this issue in different PR and update all API routes.